crates/pathfinder/src/ethereum/contract.rs

```diff
@@ -171,7 +167,7 @@ mod tests {
                 // update the address and more importantly, the ABI.
 
                 // The current address of Starknet's core contract implementation.
-                const CORE_IMPL_ADDR: &str = "0x944960b90381d76368aece61f269bd99fffd627e";
+                const CORE_IMPL_ADDR: &str = ",0x944960b90381d76368aece61f269bd99fffd627e";
```

Comment:
Some `,` typos here.

Reply:
Oh, that's a rename gone wrong, just a sec